man/systemd-gpt-auto-generator.xml

```diff
@@ -149,44 +162,49 @@
+            <entry><filename>/</filename>, <filename>/home/</filename>, <filename>/srv/</filename>, Extended Boot Loader Partition</entry>
             <entry>Partition is mounted read-only</entry>
           </row>
 
           <row>
             <entry><constant>GPT_FLAG_NO_AUTO</constant></entry>
             <entry>0x8000000000000000</entry>
-            <entry><filename>/</filename>, <filename>/srv</filename>, <filename>/home</filename></entry>
+            <entry><filename>/</filename>, <filename>/home/</filename>, <filename>/srv/</filename>, Extended Boot Loader Partition</entry>
             <entry>Partition is not mounted automatically</entry>
           </row>
 
           <row>
             <entry><constant>GPT_FLAG_NO_BLOCK_IO_PROTOCOL</constant></entry>
             <entry>0x0000000000000002</entry>
-            <entry>ESP</entry>
+            <entry>EFI System Partition (ESP)</entry>
             <entry>Partition is not mounted automatically</entry>
           </row>
         </tbody>
       </tgroup>
     </table>
 
-    <para>The <filename>/home</filename> and <filename>/srv</filename>
-    partitions may be encrypted in LUKS format. In this case, a device
-    mapper device is set up under the names
-    <filename>/dev/mapper/home</filename> and
-    <filename>/dev/mapper/srv</filename>. Note that this might create
-    conflicts if the same partition is listed in
-    <filename>/etc/crypttab</filename> with a different device mapper
-    device name.</para>
-
-    <para>Mount and automount units for the EFI System Partition (ESP) are generated on EFI systems. The ESP is mounted
-    to <filename>/boot</filename>, unless a mount point directory <filename>/efi</filename> exists, in which case it is
-    mounted there. Since this generator creates an automount unit, the mount will only be activated on-demand, when
-    accessed. On systems where <filename>/boot</filename> (or <filename>/efi</filename> if it exists) is an explicitly
-    configured mount (for example, listed in <citerefentry
-    project='man-pages'><refentrytitle>fstab</refentrytitle><manvolnum>5</manvolnum></citerefentry>) or where the
-    <filename>/boot</filename> (or <filename>/efi</filename>) mount point is non-empty, no mount units are
-    generated.</para>
+    <para>The <filename>/home/</filename> and <filename>/srv/</filename> partitions may be encrypted in LUKS
+    format. In this case, a device mapper device is set up under the names
+    <filename>/dev/mapper/home</filename> and <filename>/dev/mapper/srv</filename>. Note that this might
+    create conflicts if the same partition is listed in <filename>/etc/crypttab</filename> with a different
+    device mapper device name.</para>
+
+    <para>Mount and automount units for the EFI System Partition (ESP) are generated on EFI systems. The ESP
+    is mounted to <filename>/boot/</filename>, unless a mount point directory <filename>/efi/</filename>
```

Comment:
This text should be updated too. Maybe the paragraph about `Extended Boot Loader partition` should be first, and then this one can simply say "ESP is mounted on /boot if it not used for the other, and on /efi otherwise".